moq-pub/src/main.rs

```diff
@@ -41,7 +43,10 @@ async fn main() -> anyhow::Result<()> {
 	join_set.spawn(async move { log_viewer.run().await.context("failed to run media source") });
 
 	// TODO: generate unique namespace with UUID and/or take a command line arg
-	media_runner.announce("quic.video/moq-pub-foo", media.source()).await?;
+	if config.namespace.len() == 0 {
+		config.namespace = Uuid::new_v4().to_string();
```

Comment:
I wonder if we might want to concatenate this onto a default hostname-like prefix like `quic.video/<uuid>` to convey a bit more of what we expect the structure/semantics of namespaces to be. Fine as-is, but that's my only nit here.

Reply:
Sounds good, done.